### ISSUE TITLE
> @HongChow

### ISSUE THREAD

**Issue by tangrc** [yu4u/noise2noise#61]
> @HongChow
> 
> > training data : https://cv.snu.ac.kr/research/VDSR/train_data.zip keras=2.1.6 and tensorflow-gpu==1.5.0 but the loss is nan of inf such as : 1/1000 [..............................] - ETA: 48:13 - loss: 33339.6055 - PSNR: 5.4144 3/1000 [..............................] - ETA: 16:30 - loss: nan - PSNR: 4.5721 5/1000 [..............................] - ETA: 10:10 - loss: nan - PSNR: 4.0657 7/1000 [..............................] - ETA: 7:27 - loss: nan - PSNR: 3.8570 9/1000 [..............................] - ETA: 5:56 - loss: nan - PSNR: 3.9201 11/1000 [..............................] - ETA: 4:58 - loss: nan - PSNR: 3.9512 @yu4u could you please help with this?
> 
> I also encountered a similar problem, have you found a solution?

@tangrc hi, it seems that upgrading to the latest pytorch fixs this problem.

_Originally posted by @HongChow in https://github.com/yu4u/noise2noise/issues/60#issuecomment-1751978899_
